tests/Microsoft.Identity.Test.Unit/Kerberos/KerberosSupplementalTicketManagerTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Kerberos;
+using Microsoft.Identity.Test.Common;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using Windows.Data.Json;
+
+namespace Microsoft.Identity.Test.Unit.Kerberos
+{
+    /// <summary>
+    /// For an overview of Kerberos and testing please see: 
+    /// https://identitydivision.visualstudio.com/IdentityWiki/_wiki/wikis/IdentityWiki.wiki/20501/-AADK-AAD-Kerberos
+    /// https://identitydivision.visualstudio.com/IdentityWiki/_wiki/wikis/IdentityWiki.wiki/20601/AAD-Kerberos-for-MSAL 
+    /// </summary>
+    [TestClass]
+    public class KerberosSupplementalTicketManagerTests
+    {
+        /// <summary>
+        /// Service principal name for testing.
+        /// </summary>
+        private static readonly string _testServicePrincipalName = "HTTP/prod.aadkreberos.msal.com";
+
+        /// <summary>
+        /// Username within the ID token.
+        /// </summary>
+        private static readonly string _testClientName = "localAdmin@aadktest.onmicrosoft.com";
+
+        /// <summary>
+        /// Sample ID Token without Kerbero Service Ticket.
+        /// </summary>
+        private static readonly string _testIdToken =
+            "eyJ0eXAiOiJKV1QiLCJyaCI6IjAuQWdBQXI0R0lRckdhczBDQldEWVJOWV9fYUlLMElWSlJKck5NbXRqQW1uamszcDRzQU5NLiIsImFsZyI6IlJTMjU2"

```

Comment:
do we think that this is OK? / will be accepted by compliance?

Reply:
It's a IdToken sample and doesn't contains any sensitive information. Is that a matter?